api/app/Http/Controllers/MaterialController.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Material;
+use Illuminate\Support\Facades\DB;
+
+class MaterialController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+        $slug = $request->path();
+        $materials = DB::table('learning_materials as m')
```

Comment:
I myself would've used Eloquent since our system is not complex enough to be concerning ourselves with performance. Just a note for the future.